src/Symfony/Bridge/PhpUnit/Tests/DeprecationErrorHandler/default.phpt

```diff
@@ -0,0 +1,62 @@
+--TEST--
+Test DeprecationErrorHandler in default mode
+--SKIPIF--
+<?php if (!file_exists('DeprecationErrorHandler.php')) print "skip"; ?>
+--FILE--
+<?php
+
+putenv('SYMFONY_DEPRECATIONS_HELPER');
+
+define('PHPUNIT_COMPOSER_INSTALL', __DIR__.'/../../vendor/autoload.php');
```

Comment:
wouldn't it be better to check for ``__DIR__.'/../../vendor/autoload.php'`` instead then ? It would at least be easier to understand